src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerWithCropsValueConverter.cs

```diff
@@ -51,22 +51,27 @@ public override object ConvertIntermediateToObject(IPublishedElement owner, IPub
 
             var mediaItems = new List<MediaWithCrops>();
             var dtos = MediaPicker3PropertyEditor.MediaPicker3PropertyValueEditor.Deserialize(inter);
+            var configuration = propertyType.DataType.ConfigurationAs<MediaPicker3Configuration>();
 
             foreach (var dto in dtos)
             {
                 var mediaItem = _publishedSnapshotAccessor.PublishedSnapshot.Media.GetById(preview, dto.MediaKey);
                 if (mediaItem != null)
                 {
-                    mediaItems.Add(new MediaWithCrops
+                    var localCrops = new ImageCropperValue
                     {
-                        MediaItem = mediaItem,
-                        LocalCrops = new ImageCropperValue
-                        {
-                            Crops = dto.Crops,
-                            FocalPoint = dto.FocalPoint,
-                            Src = mediaItem.Url()
-                        }
-                    });
+                        Crops = dto.Crops,
+                        FocalPoint = dto.FocalPoint,
+                        Src = mediaItem.Url()
+                    };
+
+                    localCrops.ApplyConfiguration(configuration);
+
+                    // TODO: This should be optimized/cached, as calling Activator.CreateInstance is slow
```

Comment:
I've added this to-do, because it's also something that's done in the `BlockListPropertyValueConverter` and might have considerable performance gains by using similar techniques as in `PublishedModelFactory` (caching emitted constructor functions using `ReflectionUtilities`). Maybe @Shazwazza can give some pointers on how to best handle/implement this?